pkgs/stdenv/linux/make-bootstrap-tools-cross.nix

```diff
@@ -22,6 +22,7 @@ in lib.mapAttrs (n: make) (with lib.systems.examples; {
   mipsel-linux-gnu         = mipsel-linux-gnu;
   mips64el-linux-gnuabin32 = mips64el-linux-gnuabin32;
   mips64el-linux-gnuabi64  = mips64el-linux-gnuabi64;
+  powerpc64 = ppc64;
```

Comment:
Also a another one https://github.com/NixOS/nixpkgs/pull/258540

(working on automating the bootstrap file updates, that's why it's changing so much)

Reply:
I could change it to `powerpc64-linux-gnuabielfv2`, or I think I wait for the exact formatting of these to be decided over in that PR.

Reply:
Matched with the changes in https://github.com/NixOS/nixpkgs/pull/258540.